tensorboard/plugins/image/metadata.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Internal information about the images plugin."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+import json
+
+import tensorflow as tf
+
+
+PLUGIN_NAME = 'images'
+
+
+ImageMetadata = collections.namedtuple('ImageMetadata', ())
+
+
+def create_summary_metadata(display_name, description):
```

Comment:
I do think that this module should provide the `PLUGIN_NAME` field: that should be centralized, and one shouldn't have to depend on the actual backend plugin (`images_plugin.py`) to access it.

But I am open to deleting these two functions and letting this module expose a single constant, `PLUGIN_NAME`. What do you think of that?